win32ss/user/winsrv/consrv/condrv/console.c

```diff
@@ -531,9 +533,14 @@ ConDrvSetConsoleCP(IN PCONSOLE Console,
         return STATUS_INVALID_PARAMETER;
 
     if (OutputCP)
+    {
         Console->OutputCodePage = CodePage;
+        Console->IsCJK = IsCJKCodePage(CodePage);
```

Comment:
I think the `Console->IsCJK` should also be updated there: https://git.reactos.org/?p=reactos.git;a=blob;f=win32ss/user/winsrv/consrv/settings.c;hb=6ad4f9e7bbab0effbdb18423f842dc6ab1d1dc12#l55

Reply:
OK.